### PR TITLE
Bump to 1.2

### PR DESCRIPTION
**Pull request by RaulPPelaez**
Acked-by: RaulPPealez <raulppelaez@gmail.com>

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by RaulPPelaez** [comment]
@conda-forge-admin, please rerender

**Comment by RaulPPelaez** [comment]
@conda-forge-admin, please rerender

**Comment by RaulPPelaez** [comment]
Ready to merge!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,10 +20,10 @@ requirements:
     - python >=3.8
   run:
     - numpy
-    - openmm >=8.0.0
+    - openmm
     - python >=3.8
-    - openmm-torch >=1.0
-    - torchani >=2.2.2
+    - openmm-torch
+    - torchani
```
Comment (peastman) [inline recipe/meta.yaml]: Why did you remove the version requirements?  It still requires them.

Comment (RaulPPelaez) [reply recipe/meta.yaml]: Re-added them!

**Comment by peastman** [comment]
Looks good, thanks!
